pkg/scheduler/scheduler.go

```diff
@@ -32,7 +32,7 @@ type kv struct {
 	Value int64
 }
 
-// getNodeList gets the nodelist which satisfies the topology info
+// getNodeList gets the nodelist which satisfies the topology info, excludes cordoned nodes.
```

Comment:
Have you check if there's no code for which the `Unschedulable` nodes would not be a desired filter?
Otherwise I'd suggest either adding an unschedulable parameter, or perhaps adding a new function

Reply:
Yes @tiagolobocastro , In `lib-csi` its called only by 1 function i.e `func Scheduler`.  In `localpv-zfs` and `localpv-lvm`this function is called only once while picking node to provision volume.

Should be fine to filter this way..

Reply:
Ok, fair enough then